nifi-commons/nifi-property-utils/src/main/java/org/apache/nifi/properties/BootstrapProperties.java

```diff
@@ -29,6 +31,9 @@
     private static final String PROPERTY_KEY_FORMAT = "%s.%s";
     private static final String BOOTSTRAP_SENSITIVE_KEY = "bootstrap.sensitive.key";
 
+    // HashiCorp Vault Sensitive Property Provider configuration
+    private static final String BOOTSTRAP_SPP_HASHICORP_VAULT_PROPERTIES = "bootstrap.sensitive.props.hashicorp.vault.properties";
```

Comment:
Although naming this property clearly in challenging, it seems better to avoid `sensitive.props` as part of the property name since that prefix refers to encryption of flow properties in `nifi.properties`.  Perhaps `bootstrap.protection.hashicorp.vault.conf` or `bootstrap.protection.hashicorp.vault.properties.path`?